tomcat/src/main/java/org/apache/coyote/http11/Http11Processor.java

```diff
@@ -26,22 +42,125 @@ public void run() {
 
     @Override
     public void process(final Socket connection) {
-        try (final var inputStream = connection.getInputStream();
-             final var outputStream = connection.getOutputStream()) {
+        try (
+                final var inputStream = connection.getInputStream();
+                final var outputStream = connection.getOutputStream()
+        ) {
+            final String requestURL = parseRequestURL(inputStream);
+
+            String resource = requestURL.split("\\?")[0];
+
+            if (resource.equals("/")) {
```

Comment:
해당 첫 번째 if문과 62번째 라인의 세 번째 if문이 동일하게 `index.html`을 반환하도록 분기 처리가 이루어지고 있는데,
 저라면 조건을 통합해 처리했을 것 같더라구요!

```suggestion
            if (resource.equals("/") || resource.startsWith("/index.html")) {
```

노랑은 어떻게 생각하시나요? 수정 제안을 드리기보다 그저 생각이 궁금해서 여쭈어봐요 😸 (추후 확장성을 고려하는 등...)

Reply:
좋은데요? 반영하겠습니다 👍